packages/shared/src/components/modals/SteppedModal.tsx

```diff
@@ -83,8 +159,9 @@ function SteppedModal({
   return (
```

Comment:
For this component I'm not super happy that the auth is so baked in this thing.

Can't we have the stepped modal be plain generic, it simply steps some children regardless what they are?
Then we can have the FeedFilterModal decide what those children are?

CC: @idoshamun How do you feel about this one?

Reply:
It was part of the requirement for the auth to be baked in the onboarding so I thought maybe in the future there will be another requirement with stepped modal to have auth as well.

It would actually be so much better if we can just use the `showLogin` (our generic auth modal) instead of creating this new auth UI.

Reply:
I also didn't like the fact that it's baked in. But I think we can refactor it in the first moment we need the stepped modal to be more generic. Because it's used only in the feed filters at the moment it's not a critical point imo

Reply:
Sure let's leave it as is for now.
Whenever another tasks needs the stepped modal we can asses what changes we need.
Resolving for now.